user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Using_existing_scripts.md

```diff
@@ -64,20 +60,20 @@ For detailed info, refer to [Defining a new API](xref:UD_APIs_Define_New_API).
 > - You can keep the existing script parameters of your script, as these will be ignored when the Automation script is executed as an API through the `OnApiTrigger` entry point method.
 > - If the `Run` method stays in the script, it will still be possible to trigger the script via Cube or any other supported DataMiner module (Scheduler, Correlation, etc.).
 
-For example, to prepare the [example script](#example-script) so it can be used to define an API, the following changes are needed:
+To prepare the [example script](#example-script) so it can be used to handle API triggers, the following changes are needed:
 
 - Add the `OnApiTrigger` method with the `AutomationEntryPoint` attribute.
 - Move the actual script behavior (stopping an element) to a separate method.
 - Add a call to this separate method in both the entry point method and the normal `Run` method.
 
-This results in the script below, where the `Run` method and the entry point method both validate the input parameters and execute the `InnerRun` method that stops the element. However, the `Run` method uses the script parameters, while the entry point method uses the *Parameters* property from the `ApiTriggerInput`. Note also that with the entry point method, specific errors and success messages can be returned to the API users.
+This results in the script below, where the `Run` method and the entry point method both validate the input parameters and execute the `InnerRun` method that stops the element. However, the `Run` method uses the script parameters, while the entry point method uses the `Parameters` property from the `ApiTriggerInput`. Note also that with the entry point method, specific errors and success messages can be returned to the API users.
```

Comment:
Shouldn't the entry point method also be between '' just as for the 'run' method

Reply:
The backticks are only used to indicate the exact names of specific methods. That's why they're not used there.